RocketChat/Rocket.Chat
# feat: Add support to cancel editing message action

Proposed changes (including videos or screenshots)

BEFORE THE FIX
https://github.com/RocketChat/Rocket.Chat/assets/112304873/1178e764-966d-4d05-b379-8a2b21e91f6b

AFTER THE FIX
https://github.com/RocketChat/Rocket.Chat/assets/112304873/832d2bfc-f347-4b88-93b5-13f80893e6e8



REASON BEHIND CHANGES
Instead of a button to cancel, it is much more intuitive to guide the user to what to do. This feature is implemented in DISCORD and I took inspiration from there. If a button is needed atmost please give me the UI as it will be easy for me to implement than to design by my own.

Issue(s)
This should close issues :-
closes https://github.com/RocketChat/Rocket.Chat/issues/30821

Steps to test or reproduce

Go to Teams/Channel
Click on any of the teams/Channels where you are participant.
Try to Send a Text to the Team/Channel
Try to Edit the Text and See that there is no Cancel Button for Edit Text if the user wants to continue without editing the Text.
